tools/fq.wdl

```diff
@@ -26,7 +26,7 @@ task fqlint {
     runtime {
         disk: disk_size + " GB"
         memory: memory_gb + " GB"
-        docker: 'ghcr.io/stjudecloud/fqlib:1.0.1'
+        docker: 'ghcr.io/stjudecloud/fqlib:branch-ci-1.2.0'
```

Comment:
This seems like something we don't want to merge in, right? Was this just done for testing?

Reply:
Just for testing, already fixed.